include/deal.II/grid/tria_accessor.h

```diff
@@ -3443,6 +3449,35 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
    * @}
    */
 
+  /**
+   * Return global active cell id for an active cell.
+   */
+  types::global_cell_index
+  global_active_cell_index() const;
+
+  /**
+   * Return local index inside of the partitioner returned by
+   * global_active_cell_index_partitioner().
+   *
+   * @note For a serial Triangulation, the returned value is the same as
+   *   returned by active_cell_index().
+   */
+  types::global_cell_index
+  global_active_cell_index_local() const;
```

Comment:
I find this name a bit confusing by having `global` and `local` in the name. For what purposes do we anticipate using this index, given that it can be computed reasonably quickly by `tria.get_active_cell_index_partitioner().global_to_local(cell->global_active_cell_index())`? Of course, this would imply that we actually store the global index in the mesh (rather than the MPI-local one), and skip all the index translations in the implementation below.

Reply:
I am actually translating during setup the global indices to local indices. This function returns simply that value.

Reply:
I know, that's what I meant by "skip all the index translations" because we would just fill the global index.

Reply:
I am not sure what you are suggesting. Should we save the actual global index?

In my opinion, it is more useful to save the MPI-local indices since in normal case we would access some vectors. But I can change this.

Reply:
Yes, my intuition would have been to save the global index and only provide an access function to that variable since I see the confusion of two "MPI-local" indices, the `active_cell_index()` we already have and this one here. I admit it is not perfect because one needs to go through the index translations and access vectors with `operator()`. Anyway, this is up to discussion and I wanted to
- hear what others think and
- in case we like the more direct access to the MPI-local indices, have others participate in finding a good name for the access function.

Reply:
As discussed offline, I am now storing the global indices and have removed the function that are ending with `_local` for now.